core/CMakeLists.txt

```diff
@@ -35,6 +35,15 @@ detray_add_library( detray_core core
 target_link_libraries( detray_core
    INTERFACE covfie::core vecmem::core detray::Thrust )
 
+# Generate a version header for the project.
+configure_file( "../cmake/version.hpp.in"
```

Comment:
I don't like this. :frowning:

Just move the `version.hpp.in` file under `core/cmake`. Like I did in vecmem. I don't like to refer to a file like this.

Reply:
Ok